tests/plugins/vllm_add_dummy_model/vllm_add_dummy_model/my_gemma_embedding.py

```diff
@@ -0,0 +1,15 @@
+import torch
+
+from vllm.model_executor.models.gemma2_embedding import Gemma2EmbeddingModel
+
+
+class MyGemma2Embedding(Gemma2EmbeddingModel):
+
+    def forward(self, *args, **kwargs) -> torch.Tensor:
+        hidden_states = super().forward(*args, **kwargs)
+
+        # We assume PP isn't used in the test
```

Comment:
any technical difficulty to support PP? this code can be an example code to add custom embedding models, and I'd prefer to add pp support if possible. should be as easy as adding one branch `if hidden_states is not None`?

Reply:
> any technical difficulty to support PP? this code can be an example code to add custom embedding models, and I'd prefer to add pp support if possible. should be as easy as adding one branch `if hidden_states is not None`?

I didn't think of that. It isn't hard to support PP, I just did this simplification for the sake of the test. Let me update this.